scripts/build_doc_version_index.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
```

Comment:
Can you follow this [style guide](https://google.github.io/styleguide/shell.xml) for your bash scripts?

We're also indenting 4 spaces, thanks.

Reply:
Just to make confirm, we're indenting 4 spaces as opposed to the 2 spaces [recommended](https://google.github.io/styleguide/shell.xml?showone=Indentation#Indentation) by the style guide?

Reply:
Yes please, thanks.

Reply:
Haha, no worries, I've updated everything else to adhere as best as possible to style guide.